packages/ui-tests/cypress/e2e/designer/specialCamelRoutes/routeConfiguration.cy.ts

```diff
@@ -3,26 +3,40 @@ describe('Test for root route configuration container', () => {
     cy.openHomePage();
   });
 
+  it('Update routeConfiguration description', () => {
```

Comment:
@tplevko, I added this dedicated e2e to explain that after updating the description, the node label will change.